src/gen-api-models.ts

```diff
@@ -119,7 +113,7 @@ export function renderOperation(
   const params: { [key: string]: string } = {};
   const importedTypes = new Set<string>();
   if (operation.parameters !== undefined) {
-    operation.parameters.forEach(param => {
+    operation.parameters.forEach((param: any) => {
```

Comment:
Type declaration of 'any' loses type-safety
Try to place the right type so the compiler helps you to check if the following code is consistent to that type
Check [here](https://github.com/kogosoftwarellc/open-api/blob/18d76c68221dd2abdccfe7e18f4462ca45c40ecb/packages/openapi-types/index.ts#L92)

Reply:
this is the [type](https://github.com/kogosoftwarellc/open-api/blob/18d76c68221dd2abdccfe7e18f4462ca45c40ecb/packages/openapi-types/index.ts#L510) but is not possible to export from library

Reply:
@Undermaken iiuc the existing code already uses `(param as any)` see below so we are not really losing type information. This doesn't mean we can improve things in a future PR.

Reply:
> this is the [type](https://github.com/kogosoftwarellc/open-api/blob/18d76c68221dd2abdccfe7e18f4462ca45c40ecb/packages/openapi-types/index.ts#L510) but is not possible to export from library

If you are sure of that, you could try to convert parameters in a typed array
```suggestion
   const parameters = operation.parameters.map(
      p => p as OpenAPIV2.ParameterObject
    );
    parameters.forEach(param => {
```
and then remove all casts to any.
In this way compiler helps you to check if members accessing is done right